sceptre/strides/config/prod/github-oidc-sage-bionetworks.yaml

```diff
@@ -14,3 +14,7 @@ sceptre_user_data:
   Repositories:
     - name: "synapse-login-aws-infra"
       branches: ["dev", "prod"]
+    - name: "synapse-login-aws-infra"
```

Comment:
there's already an entry for `synapse-login-aws-infra` so I think this just needs to update the branches list. other than that, this change looks good to me.

Reply:
fixed, thanks